core/network/src/main/java/org/sopt/network/di/ApiModule.kt

```diff
@@ -0,0 +1,24 @@
+package org.sopt.network.di
+
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import org.sopt.network.api.AuthApi
+import org.sopt.network.api.ReqresApi
+import retrofit2.Retrofit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object ApiModule {
+    @Provides
+    @Singleton
+    fun provideAuthService(@Auth retrofit: Retrofit): AuthApi =
+        retrofit.create(AuthApi::class.java)
+
+    @Provides
+    @Singleton
+    fun provideReqresService(@Reqres retrofit: Retrofit): ReqresApi =
+        retrofit.create(ReqresApi::class.java)
```

Comment:
```suggestion
    @Provides
    @Singleton
    fun provideAuthService(@Auth retrofit: Retrofit): AuthApi =
        retrofit.create()

    @Provides
    @Singleton
    fun provideReqresService(@Reqres retrofit: Retrofit): ReqresApi =
        retrofit.create()
```

create에 타입 기반 생성 로직 있어서 이렇게만 해도 생성될거에요.

Reply:
와 처음 알았습니다 ! 감사드립니다 !